src/main/java/com/epam/training/array/ArrayLogic.java

```diff
@@ -1,16 +1,85 @@
 package com.epam.training.array;
 
+import java.util.Arrays;
+import java.util.Collections;
+
 public class ArrayLogic {
 
     //TODO: write it using binary search
-    public boolean exists(Array array) {
-        throw new UnsupportedOperationException();
+    public boolean searchInArray(Array array, int searchedNumber) {
+        int[] items = array.getItems();
+        //int searchResult = Arrays.binarySearch(items, searchedNumber);
```

Comment:
@judita-b Could you remove extra comments please

Reply:
I have removed the extra comments